node.gni

```diff
@@ -27,6 +27,7 @@ declare_args() {
     "deps/cjs-module-lexer/dist/lexer.js",
     "deps/undici/undici.js",
     "deps/amaro/dist/index.js",
+    "deps/corepack/dist/lib/download.js",
```

Comment:
So this would make Corepack a dependency of Node.js core? I'm not sure how I feel about that. I still have concerns about how it differs so much from the Node codebase (written in TypeScript, dependent on Yarn) and it has an uncertain future. This seems to make the project responsible for Corepack's continued development. It also seems to contradict the recent vote to remove Corepack (I understand it would become a hidden dependency and no longer available as a binary, but presumably it would be accessible via `--expose-internals`?)

We already bundle npm and it is the canonical dependency downloader; why not use that? Or more targeted dependencies of npm like Arborist.

Reply:
npm has a even more uncertain future, and is not maintained by us. It would also increase the binary size much more, but in any case, which code does the download matters very little (Corepack as a dep, copy the code from Corepack into core, another lib, etc.), what’s more important to establish is whether we want the feature

Reply:
Define “the feature”. Is it installing npm's lockfile? Yarn's or pnpm's? All of them? Something else? Does it depend on the package manager being downloaded first?